application/src/main/resources/ecc.yml

```diff
@@ -99,6 +99,17 @@ connection:
       time: 45
       unit: MINUTES
   jmx:
+    jolokia:
+      #
+      # Specifies whether the Jolokia adapter for connecting to JMX objects is enabled or not.
+      # Set to true to enable the Jolokia adapter.
+      #
+      enabled: false
+      #
+      # Defines the port used by Jolokia for accessing JMX objects.
+      # The default port for Jolokia access.
+      #
+      port: 8778
```

Comment:
Perhaps describe that these (false/8778) are the actual default values used - if nothing is set in the config file.